### PR TITLE
Disable "global" in vitest

### LINKED ISSUE #9: Disable "global" in vitest
I tried and ran into TS issues. Would love to ditch it though.

### PR DESCRIPTION
**Pull request by onemen**
closes #9

### CODE REVIEW

**Review by kentcdodds — COMMENTED** [review]
Thank you! Can you check that the expect API can still use the testing library assertions with this?

File: other/test-setup/setup-test-env.ts [hunk 1]
```diff
@@ -1,22 +1,12 @@
...
 
-declare global {
-	namespace Vi {
-		interface JestAssertion<T = any>
-			extends jest.Matchers<void, T>,
-				TestingLibraryMatchers<T, void> {}
-	}
-}
-
-expect.extend(matchers)
```
Comment (kentcdodds) [inline other/test-setup/setup-test-env.ts:18]: I think we do still need to extend expect somehow so these matchers are available at runtime.

Comment (onemen) [reply other/test-setup/setup-test-env.ts:18]: vitest

https://vitest.dev/api/expect.html#expect-extend

Comment (onemen) [reply other/test-setup/setup-test-env.ts:18]: Yes

we need `expect.extend(matchers)` back

**Comment by onemen** [comment]
now everything work

**Review by kentcdodds — APPROVED** [review]
Stellar! Thank you!

**Pull request merged**